nubis/puppet/files/nubis-taint-reap

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-PATH=/usr/local/sbin:/usr/local/bin:$PATH
+# Be explicit to be safe
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Just to be safe from the various strange distros we deal with, let's be explicit